Makefile

```diff
@@ -25,9 +25,6 @@ compile:
 
 check_contracts:
 	@cargo install cosmwasm-check
-	@cosmwasm-check --available-capabilities iterator,staking,stargate,neutron artifacts/*.wasm
+	@cosmwasm-check --available-capabilities iterator,staking,stargate,neutron,cosmwasm_1_1,cosmwasm_1_2,cosmwasm_1_3,cosmwasm_1_4,cosmwasm_2_0 artifacts/*.wasm
```

Comment:
it's for check_contracts only. Otherwise getting these:
<img width="1348" alt="image" src="https://github.com/user-attachments/assets/42a70f3a-df21-41e4-8c71-cb920fbb2275">